### PR TITLE
Using h1 tags for blog titles

### PR DESCRIPTION
**Pull request by smilegodly**
## Issue This PR Addresses
https://github.com/Seneca-CDOT/telescope/issues/768

## Type of Change

- [ ] **Bugfix**: Change which fixes an issue
- [ ] **New Feature**: Change which adds functionality
- [ ] **Documentation Update**: Change which improves documentation
- [x] **UI**: Change which improves UI

## Description

Alphabetized the allowed tags list and deleted some duplicates I found.

Changed titles to use h1 tags instead of h2

Non-title h1 tags were 2.5em so I've reduced them to 2em
Title tags were 2em so I've increased them to 2.5em
With this text-size swap the title h1 tags are 0.5em larger then the non-title h1 tags.

Here's what the text-size change to the title visually looks like

image

## Checklist

- [x] **Quality**: This PR builds and passes our npm test and works locally
- [ ] **Tests**: This PR includes thorough tests or an explanation of why it does not
- [x] **Screenshots**: This PR includes screenshots or GIFs of the changes made or an explanation of why it does not (if applicable)
- [ ] **Documentation**: This PR includes updated/added documentation to user exposed functionality or configuration variables are added/changed or an explanation of why it does not(if applicable)

### CODE REVIEW

File: src/frontend/src/components/Post/Post.jsx [hunk 1]
```diff
@@ -52,9 +52,9 @@ const Post = ({ id, html, author, url, title, date }) => {
   return (
     <Container className={classes.root}>
       <header className={classes.header}>
-        <h2 id={id} className={classes.title}>
+        <h1 id={id} className={classes.title}>
           {title}
-        </h2>
+        </h1>
```
Comment (cindyledev) [inline src/frontend/src/components/Post/Post.jsx]: opening and closing tags don't match

**Comment by c3ho** [comment]
@smilegodly feel free to rebase this and merge

**Comment by smilegodly** [comment]
@c3ho I don't think I have write permissions

File: src/backend/utils/sanitize-html.js [hunk 2]
```diff
@@ -8,38 +8,38 @@ module.exports = function(dirty) {
...
       'h4',
-      'pre',
-      'blockquote',
+      'h5',
+      'h6',
       'hr',
+      'i',
+      'img',
+      'li',
+      'nl',
```
Comment (humphd) [inline src/backend/utils/sanitize-html.js]: I'm not sure what `nl` is, seems like a previous typo.  Could you remove here, please?

File: src/backend/utils/sanitize-html.js [hunk 3]
```diff
@@ -8,38 +8,38 @@ module.exports = function(dirty) {
...
+      'ol',
+      'p',
+      'pre',
+      'strike',
```
Comment (humphd) [inline src/backend/utils/sanitize-html.js]: Related: https://github.com/Seneca-CDOT/telescope/issues/837

**Pull request merged**